app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
```

Comment:
Storing the `Person` instance in the `people` dictionary using the name as the key assumes that all names are unique. If there are duplicate names, this will cause the previous entry to be overwritten.